website/screens/components/container/code/ContainerCodePage.tsx

```diff
@@ -0,0 +1,482 @@
+import {
+  DxcFlex,
+  DxcTable,
+  DxcLink,
+  DxcParagraph,
+} from "@dxc-technology/halstack-react";
+import Code from "@/common/Code";
+import DocFooter from "@/common/DocFooter";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import StatusTag from "@/common/StatusTag";
+import TableCode, { ExtendedTableCode } from "@/common/TableCode";
+import Example from "@/common/example/Example";
+import basicUsage from "./examples/basicUsage";
+import listbox from "./examples/listbox";
+
+const backgroundTypeString = `{
+    attachment?: string;
+    clip?: string;
+    color?: CoreColorTokens;
+    image?: string;
+    origin?: string;
+    position?: string;
+    repeat?: string;
+    size?: string;
+}`;
+const borderTypeString = `BorderProperties | {
+  top?: BorderProperties;
+  right?: BorderProperties;
+  bottom?: BorderProperties;
+  left?: BorderProperties;
+}`;
+const borderPropertiesTypeString = `{
+    width?: string;
+    style?: LineStyleValues;
+    color?: CoreColorTokens;
+}`;
+const insetTypeString = `{
+    top?: string;
+    right?: string;
+    bottom?: string;
+    left?: string;
+}`;
+const outlineTypeString = `{
+    width?: string;
+    style?: LineStyleValues;
+    color?: CoreColorTokens;
+    offset?: string;
+}`;
+const overflowTypeString = `OverflowValues | 
+{ 
+    x?: OverflowValues; 
+    y?: OverflowValues;
+}`;
+
+const sections = [
+  {
+    title: "Props",
+    content: (
+      <DxcTable>
+        <tr>
+          <th>Name</th>
+          <th style={{ minWidth: "200px" }}>Type</th>
+          <th>Description</th>
+          <th>Default</th>
+        </tr>
+        <tr>
+          <td>background</td>
+          <td>
+            <ExtendedTableCode>{backgroundTypeString}</ExtendedTableCode>
+          </td>
+          <td>
+            Based on the CSS property <Code>background</Code> allows configuring
```

Comment:
I think we can add a link to our color usage page to show which are the available values for CoreColorTokens.

Reply:
Additionally, Border and Background could benefit from having links  to the CSS property documentation, maybe more background than border.

Reply:
Done!